MediaBrowser.Providers/TV/TheTVDB/TvdbEpisodeProvider.cs

```diff
@@ -193,17 +193,43 @@ private static MetadataResult<Episode> MapEpisodeToResult(EpisodeInfo id, Episod
                 });
             }
 
-            foreach (var person in episode.GuestStars)
+            // GuestStars is a weird list of names and roles
+            // Example:
+            // 1: Some Actor (Role1
+            // 2: Role2
+            // 3: Role3)
+            // 4: Another Actor (Role1
+            // ...
+            for (var i = 0; i < episode.GuestStars.Length; ++i)
             {
-                var index = person.IndexOf('(');
-                string role = null;
-                var name = person;
+                var currentActor = episode.GuestStars[i];
+                var roleStartIndex = currentActor.IndexOf('(');
 
-                if (index != -1)
+                string name = currentActor;
+                string role = "";
+                if (roleStartIndex != -1)
```

Comment:
Is this ever true? i.e can you have an actor without a role?

Reply:
Yes